src/chapters/radius-auth.xml

```diff
@@ -0,0 +1,240 @@
+                <para>Create the <filename>GUACAMOLE_HOME/extensions</filename> directory, if it
+                    does not already exist.</para>
+            </step>
+            <step>
+                <para>Copy <filename>guacamole-auth-radius-0.9.14.jar</filename> into
+                        <filename>GUACAMOLE_HOME/extensions</filename>.</para>
+            </step>
+            <step>
+                <para>Configure Guacamole to use RADIUS authentication, as described
+                    below.</para>
+            </step>
+        </procedure>
+    </section>
+    <section xml:id="guac-radius-config">
+        <title>Configuring Guacamole for RADIUS authentication</title>
+        <indexterm>
+            <primary>configuring RADIUS authentication</primary>
+        </indexterm>
+        <indexterm>
+            <primary>RADIUS authentication</primary>
+            <secondary>configuration</secondary>
+        </indexterm>
+        <para>This extension provides several configuration properties in order 
+            to communicate properly with the RADIUS server to which it needs to authenticate.  It is 
+            important that you know several key pieces of information about the RADIUS server - 
+            at a minimum, the server name or IP, the Authentication port, the authentication 
+            protocol in use by the server, and the shared secret for the RADIUS client.  If you 
+            are responsible for the RADIUS server, you'll need to properly configure these items 
+            to get Guacamole to authenticate properly.  If you're not responsible for the RADIUS 
+            server you will need to work with the administrator to get all of the necessary 
+            configuration items for the server.  These items will need to be configured in the 
+            <link linkend="initial-setup"><filename>guacamole.properties</filename></link> 
+            file.</para>
+        <variablelist>
+            <varlistentry>
+                <term><property>radius-hostname</property></term>
+                <listitem>
+                    <para>The RADIUS server to authenticate against.  If not specified, 
+                        localhost will be used.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-auth-port</property></term>
+                <listitem>
+                    <para>The RADIUS authentication port on which the RADIUS service is
+                        is listening.  If not specified, the default of 1812 will be
+                        used.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-shared-secret</property></term>
+                <listitem>
+                    <para>The shared secret to use when talking to the RADIUS server.  This
+                        parameter is required and defaults to null if not specified.</para>
```

Comment:
"Defaults to null" only has useful meaning within the internals/code of the extension. This should be rephrased to something which makes sense from the user's point of view.

Reply:
Should be better.